unittest.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 26 13:53:41 2017
+
+@author: siddhant
+"""
+
```

Comment:
I like the work you have done here, the function names are very descriptive and we have almost 100% coverage of the Point class here. Thank you for writing comments for your functions. In Python, the preferred method for writing comments about a function is to use triple quotes, """ Place comment here. """, inside the function just below the def line instead of outside the function like in java. With them inside the function the Python interpriter is able to see them and turn the comments into a docstring for the function. You can see this if you create a Point in the IPython console and then type `help(var_name)`. Other than that things are looking great. Keep up the good work.